src/pages/taehyunKim/Main/Main.js

```diff
@@ -0,0 +1,357 @@
+import React, { Component } from "react";
+import { withRouter } from "react-router-dom";
+import Comment from "./Comment";
+import "./Main.scss";
+
+class MainTaehyun extends Component {
+  constructor() {
+    super();
+    this.state = {
+      comment: "",
+      commentArr: [{ str: "" }],
+      isActiveBtn: false,
+    };
+  }
+
+  handleCommentInput = (e) => {
+    this.setState({
+      comment: e.target.value,
+    });
+  };
+
+  addCommentList = () => {
+    const comment = { str: this.state.comment };
+    this.setState({
+      commentArr: this.state.commentArr.concat(comment),
+      comment: "",
+    });
+  };
+
+  clickBtn = (e) => {
+    if (e.key === "Backspace" || this.state.comment.length === 0) {
+      return;
+    } else if (e.key === "Enter") {
+      this.addCommentList();
+    }
+  };
+
+  checkComment = () => {
+    this.state.comment.length > 0 ? this.setState({ isActiveBtn: true }) : this.setState({ isActiveBtn: false });
+  };
```

Comment:
```suggestion
  checkComment = () => {
    const isActiveBtn = this.state.comment.length > 0;
    this.setState({ isActiveBtn }); // this.setState({ isActiveBtn : isActiveBtn });
  };
```
- 변수를 활용하시면 더욱 깔끔하게 작성할 수 있습니다.
- object의 key와 value가 동일한 경우 위와 같이 작성할 수 있습니다.

Reply:
너무 유용합니다 감사드려요✅